src/main/java/ch/jalu/configme/configurationdata/ConfigurationDataBuilder.java

```diff
@@ -162,27 +163,6 @@ protected void setCommentForPropertyField(@NotNull Field field, @NotNull String
         return null;
     }
 
-    /**
-     * Sets the Field object to be accessible if needed; this makes the code support constants in Kotlin without the
-     * {@code @JvmField} annotation. This method only calls {@link Field#setAccessible} if it is needed to avoid
-     * potential issues with a security manager. Within Java itself, only {@code public static final} property fields
-     * are expected.
-     *
-     * @param field the field to process
-     */
-    protected void setFieldAccessibleIfNeeded(@NotNull Field field) {
-        try {
-            if (!Modifier.isPublic(field.getModifiers())) {
-                field.setAccessible(true);
-            }
-            // CHECKSTYLE:OFF
-        } catch (Exception e) {
```

Comment:
Challenge here again whether we want to keep catch (Exception) or if we're really happy with the new way. It might be nice to always consistently have a ConfigMe exception with some context.